tests/cli.rs

```diff
@@ -435,15 +443,17 @@ Heizoelrueckstossabdaempfung.
         // Thing itself works
         assert!(output.status.success(), "Binary execution itself failed");
 
-        // Results are correct Do not drop on panic, to keep tmpdir in place for manual
-        // inspection. Can then diff directories.
+        // Do not drop on panic, to keep tmpdir in place for manual inspection. Can then
+        // diff directories.
         check_directories_equality(baseline, candidate.path().to_owned())?;
 
         // Test was successful: ok to drop.
         drop(ManuallyDrop::into_inner(candidate));
```

Comment:
TODO: I think the crate now supports this natively, w/o the need for `ManuallyDrop`.